### ISSUE TITLE
Insufficient color contrast on empty Lockbox Entries list

### ISSUE THREAD

**Issue by kimberlythegeek** [mozilla-lockbox/lockbox-extension#554]
The text "When you add an Entry, it automatically shows up here." has insufficient color contrast against its background.

Background: `#e0e0e3`
Text: `#737373`
Expected Contrast Ratio: `4.5:1`
Contrast Ratio: `3.58`

Text should be larger, or contrast should be higher. 
Some recommended text colors: `#555555`, `#545454`, `#3e3e3e`

**Comment by changecourse** [comment]
Let's change the text color to #4A4A4F @nickbrandt 

<img width="730" alt="screen shot 2018-02-21 at 1 28 46 pm" src="https://user-images.githubusercontent.com/6164801/36503698-36bc59f2-170b-11e8-87ae-8d2837127db7.png">
